[2016-11-14 05:50:29] Espo.ERROR: Uncaught Exception PDOException: "SQLSTATE[HY000] [1045] Access denied for user 'crmuser_new'@'localhost' (using password: YES)" at C:\xampp\htdocs\espo-new\application\Espo\ORM\EntityManager.php line 138 {"exception":"[object] (PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'crmuser_new'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\espo-new\\application\\Espo\\ORM\\EntityManager.php:138)"} []
[2016-11-14 05:50:29] Espo.ERROR: Uncaught Exception PDOException: "SQLSTATE[HY000] [1045] Access denied for user 'crmuser_new'@'localhost' (using password: YES)" at C:\xampp\htdocs\espo-new\application\Espo\ORM\EntityManager.php line 138 {"exception":"[object] (PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'crmuser_new'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\espo-new\\application\\Espo\\ORM\\EntityManager.php:138)"} []
[2016-11-14 05:53:39] Espo.ERROR: Uncaught Exception PDOException: "SQLSTATE[HY000] [1045] Access denied for user 'crmuser_new'@'localhost' (using password: YES)" at C:\xampp\htdocs\espo-new\application\Espo\ORM\EntityManager.php line 138 {"exception":"[object] (PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'crmuser_new'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\espo-new\\application\\Espo\\ORM\\EntityManager.php:138)"} []
[2016-11-14 05:53:39] Espo.ERROR: Uncaught Exception PDOException: "SQLSTATE[HY000] [1045] Access denied for user 'crmuser_new'@'localhost' (using password: YES)" at C:\xampp\htdocs\espo-new\application\Espo\ORM\EntityManager.php line 138 {"exception":"[object] (PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'crmuser_new'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\espo-new\\application\\Espo\\ORM\\EntityManager.php:138)"} []
[2016-11-14 05:55:33] Espo.ERROR: Uncaught Exception PDOException: "SQLSTATE[HY000] [1045] Access denied for user 'crmuser_new'@'localhost' (using password: YES)" at C:\xampp\htdocs\espo-new\application\Espo\ORM\EntityManager.php line 138 {"exception":"[object] (PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'crmuser_new'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\espo-new\\application\\Espo\\ORM\\EntityManager.php:138)"} []
[2016-11-14 05:55:33] Espo.ERROR: Uncaught Exception PDOException: "SQLSTATE[HY000] [1045] Access denied for user 'crmuser_new'@'localhost' (using password: YES)" at C:\xampp\htdocs\espo-new\application\Espo\ORM\EntityManager.php line 138 {"exception":"[object] (PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'crmuser_new'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\espo-new\\application\\Espo\\ORM\\EntityManager.php:138)"} []
[2016-11-14 08:02:03] Espo.ERROR: E_RECOVERABLE_ERROR: Argument 4 passed to Espo\Custom\Controllers\Webhook::actionGenerate() must be an instance of Espo\ORM\Entity, none given, called in C:\xampp\htdocs\espo-new\application\Espo\Core\ControllerManager.php on line 116 and defined {"code":4096,"message":"Argument 4 passed to Espo\\Custom\\Controllers\\Webhook::actionGenerate() must be an instance of Espo\\ORM\\Entity, none given, called in C:\\xampp\\htdocs\\espo-new\\application\\Espo\\Core\\ControllerManager.php on line 116 and defined","file":"C:\\xampp\\htdocs\\espo-new\\custom\\Espo\\Custom\\Controllers\\Webhook.php","line":17,"context":{"params":{"controller":"Webhook","action":"generate"},"data":"","request":"[object] (Slim\\Http\\Request: {\"headers\":{},\"cookies\":{}})"}} []
[2016-11-14 09:37:52] Espo.ERROR: Uncaught Exception PDOException: "SQLSTATE[HY000] [1044] Access denied for user 'crmuser_new'@'localhost' to database 'real_crm'" at /home/rebelute/public_html/realcrm/application/Espo/ORM/EntityManager.php line 138 {"exception":"[object] (PDOException(code: 1044): SQLSTATE[HY000] [1044] Access denied for user 'crmuser_new'@'localhost' to database 'real_crm' at /home/rebelute/public_html/realcrm/application/Espo/ORM/EntityManager.php:138)"} []
[2016-11-14 09:37:52] Espo.ERROR: Uncaught Exception PDOException: "SQLSTATE[HY000] [1044] Access denied for user 'crmuser_new'@'localhost' to database 'real_crm'" at /home/rebelute/public_html/realcrm/application/Espo/ORM/EntityManager.php line 138 {"exception":"[object] (PDOException(code: 1044): SQLSTATE[HY000] [1044] Access denied for user 'crmuser_new'@'localhost' to database 'real_crm' at /home/rebelute/public_html/realcrm/application/Espo/ORM/EntityManager.php:138)"} []
